fizzbuzz.rb

```diff
@@ -1,4 +1,12 @@
-(1..100).each do |number|
-  result = number % 3 == 0 ? 'fizz' : (number % 5 == 0 ? 'buzz' : number)
-  puts result
+(1..100).each do |n|
```

Comment:
alan please be more. descriptive in your loop variable name, change it for `number`

Reply:
done